.github/CODEOWNERS

```diff
@@ -15,14 +15,22 @@
 # Administrata and CIs
 *.toml  @larsoner
 *.yml  @larsoner
-*.rst  @larsoner
+*.rst  @larsoner @peternewman
 *.cfg  @larsoner
+codespell.1.include @peternewman
```

Comment:
Did you intentionally miss this one @larsoner ?

Reply:
```suggestion
codespell.1.include @larsoner @peternewman
```